docs/proposals/hpa-v2.md

```diff
@@ -0,0 +1,242 @@
+Horizontal Pod Autoscaler with Arbitary Metrics
```

Comment:
Can we keep the name `Custom Metrics`? I think we've been using it for quite some already and adding new term `Arbitrary Metric` might be confusing.